INSTALL.md

```diff
@@ -103,8 +108,257 @@ docker container stop elasticsearch_test; docker container rm elasticsearch_test
+# publish will run the server from the host port 8080.
+# name will give the container a handy name for referencing later.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+The server should now be available at [localhost:8080](http://localhost:8080).
+
+### Configure the server
+
+The server will use test data and a test configuration by default.
+This can be changed in several different ways.
+
+One is to `git clone` the repository locally and bind the repository folder to the `/app` folder in the container:
+
+```shell
+# volume will bind the local version of `optimade-python-tools` to the container.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    --volume /path/to/optimade-python-tools:/app \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+Where you should change the `/path/to/optimade-python-tools` to the full local path to the repository - or use `$PWD` if you are running this command from the root of the cloned repository on a Unix system.
+Equivalently, `%cd%` should work on Windows.
+
+In this setup you can change the repository root file `optimade_config.json` with the appropriate information.
+E.g., if you do not wish to use the test data you can add the `insert_test_data` key with a value of `false`.
+
+Another option is to _not_ `git clone` the repository locally, but instead only generate a JSON or YAML file somewhere, where the directory of its location is bound to another location in the container that is not used, e.g., `/config` or similar.
+As an example, let us say we generate `config.yml` locally, with the full path `/home/user/optimade/config.yml`.
+Then we can start the server with the following command:
+
+```shell
+# volume will bind the local directory to an unused folder in the container.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    --volume /home/user/optimade:/config \
+    --env OPTIMADE_CONFIG_FILE=/config/config.yml \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+As shown, it is necessary to update the environment variable `OPTIMADE_CONFIG_FILE` within the container to point to the new internal path to the config file.
+By default, this environment variable points to `/app/optimade_config.json`.
+
+This also reveals another way of configuring the server: Use environment variables when running the container that supplies the values that would otherwise be supplied from the configuration file.
+
+The `docker run` command even has the opportunity to pass a path to a file containing a list of environment variables (`--env-file /path/to/env_file`), if you wish to configure the server in this way instead of through the standard configuration file.
+
+### **Example**: Multiple container services in closed network
```

Comment:
This is useful stuff but the install docs are now very long. Could this (and perhaps some subset of the above notes) be moved to a `Deployment` section of the docs?

Reply:
Yeah, I was thinking about that as well. Just didn't have the overview to do it in the first run through. I'll try to update it, moving it around, when I get time.

Reply:
Done.